packages/devtools_app/lib/src/scaffold.dart

```diff
@@ -78,7 +79,7 @@ class DevToolsScaffold extends StatefulWidget {
         horizontalPadding.left,
         isEmbedded() ? 2.0 : 16.0,
         horizontalPadding.right,
-        0.0,
+        isEmbedded() ? 0.0 : 8.0,
```

Comment:
while you're here, can we use defaultSpacing and denseSpacing for 16.0 and 8.0

Reply:
done